packages/beta/src/api/alerts/alertsApi.ts

```diff
@@ -50,10 +52,14 @@ export class AlertsAPI extends BaseResourceAPI<Alert> {
     return this.createEndpoint(items);
   };
 
-  public list = async (filter?: AlertFilterQuery) => {
-    return this.listEndpoint<AlertFilterQuery>(
+  public list = async (filter?: AlertFilterQuery, sort?: AlertSortQuery) => {
```

Comment:
why not just use a single query property for this same as others?
```
export interface AlertFilterQuery extends FilterQuery {
  filter?: AlertFilter;
  sort?: WhateverSortTypeHere
}
```

Reply:
That's what i tried initially, but that didn't work for me because listEndpoint() expected two parameters, and not three, so i combined sort and filter so that i avoided changing listEndpoint()